pkg/detectors/gitlab/v2/gitlab_v2.go

```diff
@@ -28,14 +28,12 @@ func (Scanner) DefaultEndpoint() string { return "https://gitlab.com" }
 
 var (
 	defaultClient = common.SaneHttpClient()
-	keyPat        = regexp.MustCompile(detectors.PrefixRegex([]string{"gitlab"}) + `\b(glpat-[a-zA-Z0-9\-=_]{20,22})\b`)
+	keyPat        = regexp.MustCompile(`\b(glpat-[a-zA-Z0-9\-=_]{20,22})\b`)
 )
 
 // Keywords are used for efficiently pre-filtering chunks.
 // Use identifiers in the secret preferably, or the provider name.
-func (s Scanner) Keywords() []string {
-	return []string{"gitlab"}
-}
+func (s Scanner) Keywords() []string { return []string{"glpat"} }
```

Comment:
It may be best to use the full `glpat-`, just so an alphanumeric doesn't accidentally trigger this or something.